e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/bank-request-date.spec.js

```diff
@@ -5,7 +5,7 @@ import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../e2e-f
 import caseDealPage from '../../pages/caseDealPage';
 import { backLink, cancelLink, continueButton, errorSummary } from '../../partials';
 import bankRequestDatePage from '../../pages/deal-cancellation/bank-request-date';
-import dateConstants from '../../../../../e2e-fixtures/dateConstants';
+import { today, todayYear, twelveMonthsOneDayAgo, twelveMonthsOneDay } from '../../../../../e2e-fixtures/dateConstants';
```

Comment:
twelveMonthsOneDayAgo and twelveMonthsOneDay aren't currently exported from the `dateConstants.js` file (why this test is failing at the moment!) - I think add to the long list of module exports in that file, and I'll try get in a refactor for that long list after this PR

Reply:
Thanks @BethThomas141 that's interesting, i'd expect linting to pick up that `twelveMonthsOneDayAgo` and `twelveMonthsOneDay` are undefined